pkg/reconciler/builder/builder_test.go

```diff
@@ -56,6 +58,7 @@ func testBuilderReconciler(t *testing.T, when spec.G, it spec.S) {
 				Tracker:            fakeTracker,
 				ClusterStoreLister: listers.GetClusterStoreLister(),
 				ClusterStackLister: listers.GetClusterStackLister(),
+				K8sClient:          k8sfakeClient,
 			}
 			return r, rtesting.ActionRecorderList{fakeClient}, rtesting.EventList{Recorder: record.NewFakeRecorder(10)}
```

Comment:
make sure k8s client is being recorded:

`rtesting.ActionRecorderList{fakeClient, k8sfakeClient}`